fondat/aws/bedrock/resources/agents.py

```diff
@@ -0,0 +1,71 @@
+"""Root resource for Bedrock agents."""
+
+from collections.abc import Iterable
+from typing import Any
+
+from fondat.aws.client import Config, wrap_client_error
+from fondat.pagination import Cursor
+from fondat.resource import resource
+from fondat.security import Policy
+
+from ..clients import agent_client
+from ..decorators import operation
+from ..pagination import decode_cursor, paginate
+from .agent import AgentResource
+
+
+@resource
+class AgentsResource:
+    """
+    Resource for listing all Bedrock agents and accessing specific ones.
+    """
+
+    def __init__(
+        self,
+        *,
+        config_agent: Config | None = None,
+        config_runtime: Config | None = None,
+        policies: Iterable[Policy] | None = None,
+    ):
+        self.config_agent = config_agent
+        self.config_runtime = config_runtime
+        self.policies = policies
+
+    @operation(method="get", policies=lambda self: self.policies)
+    async def get(self, *, max_results: int | None = None, cursor: Cursor | None = None) -> Any:
```

Comment:
Minor : I don't think you need to specify the method name in the annotation since you're using `get` as the actual function name.

Also shouldn't we be returning an array of Agents or an array of `AgentResource` rather than `Any` ?